bower.json

```diff
@@ -26,7 +26,8 @@
     "angular-ui-ace": "^0.2.3",
     "angular-ui-select": "^0.19.8",
     "angular-bootstrap": "^2.5.0",
-    "angular-ckeditor": "^1.0.3"
+    "angular-ckeditor": "^1.0.3",
+    "ckeditor": "^4.8.0"
```

Comment:
This will have negative implications on our renderer size since people will now need to include ALL ckeditor files regardless if they need it or now. Previously, we have told people that if you want ckeditor, then just include it from CDN within your application. Not sure why we cannot keep doing that.